mne/tests/test_epochs.py

```diff
@@ -1629,19 +1629,41 @@ def test_saved_fname_no_splitting(
     assert not split_1_fpath.is_file()
 
 
+@pytest.mark.parametrize(
+    "epochs_to_split",
+    [
+        ("3MB", 18, False, False, 3),
+        pytest.param(("2GB", 18, False, False, 1), marks=pytest.mark.xfail),
```

Comment:
Why is there an `xfail`? Shouldn't these be working now (but failing on `main`)? (please add a comment if it indeed should be `xfail`)

Reply:
Yes, it should be xfail. Just wanted to reflect the current state of things in tests. Making this test pass requires more changes than using `validate=True` in `_construct_bids_filename` and might be a subject for another PR. I felt like for now fixing it for multiple splits is good enough.